### PR TITLE
Add HDMF to PyNWB

### PR DESCRIPTION
**Pull request by ajtritt**
## Motivation

HDMF has been moved into its own package.

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 import traceback
 import unittest2 as unittest
 
-flags = {'form': 1, 'pynwb': 2, 'integration': 3, 'example': 4}
+flags = {'pynwb': 2, 'integration': 3, 'example': 4}
```
Comment (oruebel) [inline test.py:17]: Very minor, nitpicky thing. Should this change to ``flags = {'pynwb': 1, 'integration': 2, 'example': 3}`` just to have the numbering of options start at 1. Just ignore if this causes trouble.

**Review by oruebel — APPROVED** [review]
The changes all look good to me. I think this can be merged as is.

File: test.py (hunk 1 above)
Comment (ajtritt) [reply test.py:17]: I didn't see this until after already merged... I'll get it on the next fix

Comment (oruebel) [reply test.py:17]: No problem. Its such a minor thing anyways.

**Pull request merged**